src/Engine/Loader/Loader.hpp

```diff
@@ -8,6 +8,7 @@
 #include "Library.hpp"
 
 #include <cstdint>
+#include <iostream>
```

Comment:
iostream isn't used in Loader.hpp, and it's already included in the .cpp

Reply:
This can be removed